recipes-devtools/nsight-systems/nsight-systems-cli_2020.5.3.17-1.bb

```diff
@@ -0,0 +1,53 @@
+SUMMARY = "NVIDIA Nsight Systems is a multi-core CPU sampling profiler that \
+    provides an interactive view of captured profiling data, helping improve \
+    overall application performance."
+HOMEPAGE = "https://developer.nvidia.com/nsight-systems"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = " \
+    file://opt/nvidia/nsight-systems-cli/2020.5.3/NVIDIA_SLA.pdf;md5=f7646eea4f29c4f708ea0aa5bc6778a6 \
+"
+
+inherit nvidia_devnet_downloads
+
+SRC_URI = "${NVIDIA_DEVNET_MIRROR}/nsight-systems-cli-2020.5.3_${PV}_arm64.deb;subdir=nsight-systems-cli"
+SRC_URI[sha256sum] = "8a1d1bc76ee4a88793e1df26214e0df9aa69e0b435f9e79acdca84a41e7b3c8f"
+
+S = "${WORKDIR}/nsight-systems-cli"
+B = "${S}"
+
+COMPATIBLE_MACHINE = "tegra"
+
+do_configure() {
+    :
+}
+
+do_compile() {
+    :
+}
+
+do_install() {
+    install -d ${D}/opt/nvidia/nsight-systems-cli/2020.5.3/target-linux-armv8
+    cp -R --preserve=mode,timestamps,links --no-dereference ${B}/opt/nvidia/nsight-systems-cli/2020.5.3/target-linux-armv8/* ${D}/opt/nvidia/nsight-systems-cli/2020.5.3/target-linux-armv8/
```

Comment:
You can probably install the whole `target-linux-armv8` directory insteady of using cp here, unless there's just a few things you need out of it.